docs/zoo/index.md

```diff
@@ -26,27 +26,31 @@ The model definitions and the train loops are available in the [Larq Zoo reposit
+| [QuickNetLarge](/zoo/api/sota/#quicknetlarge) | 62.7 %         | 84.0 %          |  4.49 MB | 27.6 ms                    |
+| [QuickNetXL](/zoo/api/sota/#quicknetxl)       | 67.0 %         | 87.3 %          |  6.22 MB | 48.1 ms                    |
 
 The [`literature`](/zoo/api/literature/) submodule contains the following models:
 
-| Model                                                                   | Top-1 Accuracy | Top-5 Accuracy | Parameters | Memory   |
-| ----------------------------------------------------------------------- | -------------- | -------------- | ---------- | -------- |
-| [RealToBinaryNet](/zoo/api/literature/#realtobinarynet)                 | 65.0 %         | 85.7 %         | 11 995 624 | 5.13 MB  |
-| [BinaryDenseNet45](/zoo/api/literature/#binarydensenet45)               | 64.6 %         | 85.2 %         | 13 889 512 | 7.35 MB  |
-| [BinaryDenseNet37Dilated](/zoo/api/literature/#binarydensenet37dilated) | 64.3 %         | 85.2 %         | 8 702 248  | 5.13 MB  |
-| [BinaryDenseNet37](/zoo/api/literature/#binarydensenet37)               | 62.9 %         | 84.2 %         | 8 702 248  | 5.13 MB  |
-| [MeliusNet22](/zoo/api/literature/#meliusnet22)                         | 62.4 %         | 83.9 %         | 6 944 584  | 3.88 MB  |
-| [BinaryDenseNet28](/zoo/api/literature/#binarydensenet28)               | 60.9 %         | 82.8 %         | 5 131 496  | 4.04 MB  |
-| [BinaryResNetE18](/zoo/api/literature/#binaryresnete18)                 | 58.3 %         | 80.8 %         | 11 689 640 | 4.00 MB  |
-| [Bi-Real Net](/zoo/api/literature/#birealnet)                           | 57.5 %         | 79.8 %         | 10 985 472 | 4.00 MB  |
-| [DoReFaNet](/zoo/api/literature/#dorefanet)                             | 53.4 %         | 76.5 %         | 62 394 440 | 22.80 MB |
-| [XNOR-Net](/zoo/api/literature/#xnornet)                                | 45.0 %         | 69.2 %         | 62 387 104 | 22.77 MB |
-| [Binary AlexNet](/zoo/api/literature/#binaryalexnet)                    | 36.3 %         | 61.5 %         | 61 848 720 | 7.45 MB  |
+| Model                                                                   | Top-1 Accuracy | Top-5 Accuracy | Memory   | Pixel 1 latency (1 thread) |
+| ------------------------------------------------------------------------| -------------- | -------------- | -------- | ---------------------------|
+| [RealToBinaryNet](/zoo/api/literature/#realtobinarynet)                 | 65.0 %         | 85.7 %         | 5.13 MB  | 51.3 ms                    |
+| [BinaryDenseNet45](/zoo/api/literature/#binarydensenet45)               | 64.6 %         | 85.2 %         | 7.35 MB  | 138.5 ms                   |
+| [BinaryDenseNet37Dilated](/zoo/api/literature/#binarydensenet37dilated) | 64.3 %         | 85.2 %         | 5.13 MB  | 182.9 ms                   |
+| [BinaryDenseNet37](/zoo/api/literature/#binarydensenet37)               | 62.9 %         | 84.2 %         | 5.13 MB  | 102.2 ms                   |
```

Comment:
Do you know what's going on with the dilated dense net? I am surprised that it is so much slower compared to the non-dilated version.

Reply:
Not sure why this is the case, the converted tflite file looks okay in Netron. Here are the summaries of the per-op benchmarks:
```
# BinaryDenseNet37
============================== Top by Computation Time ==============================
	             [node type]	          [start]	  [first]	 [avg ms]	     [%]	  [cdf%]	  [mem KB]	[times called]	[Name]
	                 CONV_2D	            0.000	   18.133	   18.118	 17.744%	 17.744%	     0.000	        1	[binary_densenet37/activation/Relu;binary_densenet3]:0
	              LceBconv2d	           47.337	    7.172	    7.188	  7.040%	 24.784%	     0.000	        1	[binary_densenet37/quant_conv2d_5/Conv2D;binary_den1]:24
	              LceBconv2d	           38.754	    6.224	    6.223	  6.095%	 30.879%	     0.000	        1	[binary_densenet37/quant_conv2d_4/Conv2D;binary_den1]:20
	                 CONV_2D	           58.728	    6.173	    6.160	  6.033%	 36.912%	     0.000	        1	[binary_densenet37/conv2d_1/Conv2D2]:30
	              LceBconv2d	           31.863	    5.001	    4.970	  4.868%	 41.780%	     0.000	        1	[binary_densenet37/quant_conv2d_3/Conv2D;binary_den1]:16
	              LceBconv2d	           26.244	    3.997	    3.941	  3.859%	 45.639%	     0.000	        1	[binary_densenet37/quant_conv2d_2/Conv2D;binary_den1]:12
	                 CONV_2D	           84.441	    3.380	    3.382	  3.313%	 48.952%	     0.000	        1	[binary_densenet37/conv2d_2/Conv2D2]:67
	              LceBconv2d	           80.237	    2.677	    2.710	  2.654%	 51.606%	     0.000	        1	[binary_densenet37/quant_conv2d_13/Conv2D;binary_de1]:61
	              LceBconv2d	           22.260	    2.724	    2.700	  2.645%	 54.251%	     0.000	        1	[binary_densenet37/quant_conv2d_1/Conv2D;binary_den1]:8
	              LceBconv2d	           76.947	    2.356	    2.391	  2.342%	 56.593%	     0.000	        1	[binary_densenet37/quant_conv2d_12/Conv2D;binary_de1]:57

Number of nodes executed: 151
============================== Summary by node type ==============================
	             [Node type]	  [count]	  [avg ms]	    [avg %]	    [cdf %]	  [mem KB]	[times called]
	              LceBconv2d	       32	    49.103	    48.125%	    48.125%	     0.000	       32
	                 CONV_2D	        4	    29.679	    29.088%	    77.213%	     0.000	        4
	                     ADD	       36	     6.892	     6.755%	    83.968%	     0.000	       36
	                     MUL	       36	     6.542	     6.412%	    90.379%	     0.000	       36
	           CONCATENATION	       32	     5.918	     5.800%	    96.180%	     0.000	       32
	             MAX_POOL_2D	        4	     3.183	     3.120%	    99.299%	     0.000	        4
	         FULLY_CONNECTED	        1	     0.363	     0.356%	    99.655%	     0.000	        1
	                    RELU	        3	     0.316	     0.310%	    99.965%	     0.000	        3
	         AVERAGE_POOL_2D	        1	     0.020	     0.020%	    99.984%	     0.000	        1
	                 SOFTMAX	        1	     0.015	     0.015%	    99.999%	     0.000	        1
	                 RESHAPE	        1	     0.001	     0.001%	   100.000%	     0.000	        1

# BinaryDenseNet37Dilated
============================== Top by Computation Time ==============================
	             [node type]	          [start]	  [first]	 [avg ms]	     [%]	  [cdf%]	  [mem KB]	[times called]	[Name]
	                 CONV_2D	          139.584	   25.027	   25.093	 13.769%	 13.769%	     0.000	        1	[binary_densenet37_dilated/conv2d_3/Conv2D2]:116
	                 CONV_2D	            0.000	   19.894	   18.130	  9.948%	 23.716%	     0.000	        1	[binary_densenet37_dilated/activation/Relu;binary_d]:0
	                 CONV_2D	           83.999	   12.746	   12.769	  7.006%	 30.722%	     0.000	        1	[binary_densenet37_dilated/conv2d_2/Conv2D2]:65
	              LceBconv2d	           47.395	    7.238	    7.194	  3.947%	 34.670%	     0.000	        1	[binary_densenet37_dilated/quant_conv2d_5/Conv2D;bi1]:24
	              LceBconv2d	           38.794	    6.243	    6.234	  3.421%	 38.090%	     0.000	        1	[binary_densenet37_dilated/quant_conv2d_4/Conv2D;bi1]:20
	                 CONV_2D	           58.778	    6.163	    6.163	  3.382%	 41.472%	     0.000	        1	[binary_densenet37_dilated/conv2d_1/Conv2D2]:30
	              LceBconv2d	           31.892	    4.936	    4.969	  2.726%	 44.199%	     0.000	        1	[binary_densenet37_dilated/quant_conv2d_3/Conv2D;bi1]:16
	              LceBconv2d	          133.960	    4.188	    4.166	  2.286%	 46.484%	     0.000	        1	[binary_densenet37_dilated/quant_conv2d_25/Conv2D;b1]:112
	              LceBconv2d	           26.275	    3.914	    3.943	  2.164%	 48.648%	     0.000	        1	[binary_densenet37_dilated/quant_conv2d_2/Conv2D;bi1]:12
	              LceBconv2d	          128.737	    3.911	    3.906	  2.143%	 50.791%	     0.000	        1	[binary_densenet37_dilated/quant_conv2d_24/Conv2D;b1]:108

Number of nodes executed: 147
============================== Summary by node type ==============================
	             [Node type]	  [count]	  [avg ms]	    [avg %]	    [cdf %]	  [mem KB]	[times called]
	              LceBconv2d	       32	    83.766	    45.979%	    45.979%	     0.000	       32
	                 CONV_2D	        4	    62.153	    34.116%	    80.095%	     0.000	        4
	                     ADD	       36	    11.731	     6.439%	    86.534%	     0.000	       36
	                     MUL	       36	    11.064	     6.073%	    92.607%	     0.000	       36
	           CONCATENATION	       32	    10.057	     5.520%	    98.127%	     0.000	       32
	             MAX_POOL_2D	        2	     2.573	     1.412%	    99.539%	     0.000	        2
	         FULLY_CONNECTED	        1	     0.316	     0.173%	    99.713%	     0.000	        1
	         AVERAGE_POOL_2D	        1	     0.267	     0.147%	    99.859%	     0.000	        1
	                    RELU	        1	     0.240	     0.132%	    99.991%	     0.000	        1
	                 SOFTMAX	        1	     0.014	     0.008%	    99.999%	     0.000	        1
	                 RESHAPE	        1	     0.002	     0.001%	   100.000%	     0.000	        1
```
So both the `BConv` and the regular convolutions get a lot slower.

Reply:
thanks for the table, I was just currious if this is due to an obvious issue during conversion that would be visible in netron.